debian/helpers/build-manpages.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# This script generates all the manpages based on the upstream scripts. They
+# used to be generated during build time, but now require the download of some
+# modules not present in the vendor directory (upstream removed
+# documentation-only dependencies).
+# Execute this script right after importing a new upstream version to update
+# the manpages in debian/manpages.
+
+set -eux
+
+# Create temporary directory to build manpages
+build_dir=$(mktemp -d -t docker-cli-docsgen.XXXXXXXXXX)
+
+# Define function to remove the build directory when the script exits
+function clean {
+  rm -rf "${build_dir}"
+}
+trap clean EXIT
+
+# Copy the CLI code to the build directory
+cp -r cli/ "${build_dir}"
+pushd "${build_dir}"/cli
+
+# Check if golang-1.20-go is installed
+if dpkg -l golang-1.20-go >/dev/null 2>&1; then
```

Comment:
Yes, I tried to lock to Go 1.20 to avoid potential issues. For instance, we needed to update it in some releases because we built with a different Golang version.